spack/repo/packages/py-pyprecice/package.py

```diff
@@ -18,7 +18,7 @@ class PyPyprecice(PythonPackage):
     maintainers = ["ajaust", "BenjaminRueth"]
 
     # Always prefer final version of release candidate
-    version("develop", branch="{{ branch }}")
```

Comment:
Not needed anymore, but I guess `package.py` should have been called `package.py_template` or so in earlier versions.

Reply:
Yes. This would have avoided some confusion in the past.